src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -70,7 +85,9 @@ public string PackageLoadStateTooltip
 
                 switch (Model.LoadState.State)
                 {
-                    case PackageLoadState.StateTypes.Unloaded: return Resources.PackageStateUnloadedTooltip;
+                    case PackageLoadState.StateTypes.Unloaded:
+                        return dynamoModel.PreferenceSettings.PackageDirectoriesToUninstall.Contains(Model.RootDirectory) ?
+                           Resources.PackageStateManuallyUnloadedTooltip : Resources.PackageStateAutomaticallyUnloadedTooltip;
```

Comment:
Could you explain what this i doing? I find the descriptions automatically and manually unloaded confusing.

Reply:
I removed this...IT was an attempt to distinguish between implicit and explicit Unload at UI level